assets/js/main.js

```diff
@@ -266,95 +265,43 @@ function modal() {
     });
 }
 
+function elasticSearch(query, callback) {
+    var base_url = 'https://advisorysg.ent.ap-southeast-1.aws.found.io';
+    var engine = 'ghost';
+    var search_only_key = 'search-brya3eig6n5g9ybimkw3o9u3';
+
+    var oReq = new XMLHttpRequest();
+    var payload = { query: query };
+    oReq.open(
+        'POST',
+        base_url + '/api/as/v1/engines/' + engine + '/search.json'
+    );
+    oReq.addEventListener('load', callback);
+    oReq.setRequestHeader('Content-Type', 'application/json');
+    oReq.setRequestHeader('Authorization', 'Bearer ' + search_only_key);
```

Comment:
Could we use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for the lines here?